packages/flutter/test/widgets/shadow_test.dart

```diff
@@ -93,9 +93,10 @@ void main() {
       matchesGoldenFile('shadow.PhysicalModel.enabled.png'),
     );
     debugDisableShadows = true;
-  }, skip: isBrowser);
+  });
 
-  testWidgets('Shadows with PhysicalShape', (WidgetTester tester) async {
+  for (int elevation in kElevationToShadow.keys) {
```

Comment:
It looks like it ended up here?

Reply:
Explained in another comment. Resolving this since it is commenting on a temporary change.